tools/trainerproc/main.c

```diff
@@ -16,8 +16,9 @@
 
 #define MAX_TRAINER_AI_FLAGS 32
 #define MAX_TRAINER_ITEMS 4
-#define PARTY_SIZE 6
+#define PARTY_SIZE 64
```

Comment:
can we add some prefix to party_size? I understand it is in preproc but I would be surprised why this constant has 64 when we search for it.

Reply:
The limit of 64 is right now arbitrary, the "true" limit is 255.
But a better name wouldn't hurt.